packages/react-query-devtools/src/useSyncExternalStore.native.ts

```diff
@@ -0,0 +1,3 @@
+// Temporary workaround due to an issue with react-native uSES - https://github.com/TanStack/query/pull/3601
+// @ts-ignore
+export { useSyncExternalStore } from 'use-sync-external-store/shim/index.native.js'
```

Comment:
This doesn't work since the code is bundled. Since rollup only sees `useSyncExternalStore.ts`, it just bundle it into `build/lib/index.mjs` and `build/lib/index.js`.

Reply:
How can we fix it?

Reply:
I guess we could create a proxy package with proper exports so we would not need to think about bundling.

But there are two more `.native.` files in source. One for logger and one for batched updates.

The logger is not needed anymore as someone mentioned in some ticket. And even if so it can be configured with QueryClient.

Not sure about batched updates though.